gridt/src/api/model/userLastUpdate.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * The Gridt Network API
+ * Interact with the gridt network via this API. It provides several endpoints for creating and subscribing (to) movements and switching leaders.
+ *
+ * OpenAPI spec version: 0.0.1
+ * Contact: robin@gridt.org
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+/**
+ * This is the last update that the user has given in this movement.
+ */
+export interface UserLastUpdate { 
+    /**
+     * Date time of the last update of the user.
+     */
+    timeStamp?: string;
```

Comment:
The timeStamp being a string... Not checking if it is actually a timestamp might be problematic in the future.